### ISSUE TITLE
authentication failed, email not fetched

### ISSUE THREAD

**Issue by sapkal-manish** [chirag04/mail-listener2#41]
It is giving me error like this. I have double check for my username/email and password.
I am getting following error : [Error: Please log in via your web browser: http://support.google.com/mail/accounts/bin/answer.py?answer=78754 (Failure)] textCode: 'ALERT', source: 'authentication' }
imapDisconnected

Should I have to configure my gmail account?

**Comment by chirag04** [comment]
See if imap is enabled in your gmail account.

Check: http://www.howtogeek.com/174837/how-to-add-your-gmail-account-to-outlook-2013-using-imap/

**Comment by sapkal-manish** [comment]
Yes @chirag04, I have enabled it, but still showing me same error. One another surprising things happens with Thunderbird. It's also showing me "username or password is invalid", but I am sure I am entering valid username and password.

**Comment by sapkal-manish** [comment]
When I have tried another account. It's connected, but when new mail come, thunderbird mail show new unread mail, but mail-listener2 cann't.

**Comment by parthasharathi** [comment]
In my account setting(https://myaccount.google.com/security) I just enabled 'Allow less secure apps' to ON .  Its working fine. Now my question is there any other option?
